libs/barista-components/sunburst-chart/tsconfig.json

```diff
@@ -0,0 +1,10 @@
+{
+  "extends": "../../../tsconfig.json",
+  "compilerOptions": {
+    "types": ["node", "jest"]
+  },
+  "include": [
+    "**/*.ts",
+    "../../../apps/dev/src/sunburst-chart/sunburst-chart-test-data.ts"
```

Comment:
We cannot add files outside the library folder to it, this breaks the dependency graph of nx in the long run. 
If you need test data for tests it is ok to duplicate it. 
```suggestion
```